### PR TITLE
Miscellaneous fixes

### PR DESCRIPTION
**Pull request by akx**
* Spell check
* Some of the "JSON" responses weren't really JSON
* Markdown tables were occasionally misformatted

### CODE REVIEW

**Review by jrasanen — APPROVED** [review]
+1

**Comment by jrasanen** [comment]
Thank you

File: docs/README.md [hunk 1]
```diff
@@ -20,56 +20,58 @@ Thank you!
...
-HMAC is calculated from the http requests body payload using the signing secret. Header key's must be sorted alphabetically and included in hmac calculation.
+HMAC is calculated from the HTTP request's body payload using the signing secret. Header keys must be sorted alphabetically, lower-cased and included in HMAC calculation.
 
-* Merchant ID needs to be in `Checkout-Account`
-* Algorithm needs to be in `Checkout-Algorithm` (sha256 or sha512)
-* Calculated HMAC needs to be send in each request in http-header named `Checkout-Signature`
-* Method header `Checkout-Method` needs to be set to `POST, GET OR DELETE`
+* The merchant ID needs to be in `Checkout-Account`
+* The algorithm needs to be in `Checkout-Algorithm` (SHA256 or SHA512)
+* The calculated HMAC needs to be sent in each request in the HTTP header `Checkout-Signature`
```
Comment (mikaturunen) [inline docs/README.md]: Wasn't it `signature` field?

File: docs/README.md [hunk 2]
```diff
@@ -20,56 +20,58 @@ Thank you!
...
-checkout-algorithm:sha512\n
+checkout-algorithm:SHA512\n
 checkout-method:POST\n
 checkout-nonce:1234\n
 checkout-timestamp:2018-07-05T11:19:25.950Z\n
 REQUEST BODY
 ```
 
-These are passed to hmac function which uses SHA512 algorithm. Carriage returns (\r) should not be used, only line feed (\n).
+These are passed to the HMAC function which uses SHA512 algorithm. Carriage returns (\r) should not be used, only line feed (\n).
```
Comment (mikaturunen) [inline docs/README.md]: No conditionals.

So no "should", rather say:

Do not use (\r) carriage returns, only line feed (\n).

For the implemented we are the only and single source of truth so conditionals cause confusion.

Comment (mikaturunen) [reply docs/README.md]: Or taking it even further:
Only use line feed (\n).

File: docs/README.md [hunk 3]
```diff
@@ -79,7 +81,7 @@ const SECRET = 'SAIPPUAKAUPPIAS';
 
 const headers = {
   'checkout-account': ACCOUNT,
-  'checkout-algorithm': 'sha256',
+  'checkout-algorithm': 'SHA256',
```
Comment (mikaturunen) [inline docs/README.md]: ..which one is it? SHA or sha? Juho commented on this, I think.

File: docs/README.md [hunk 4]
```diff
@@ -6,7 +6,7 @@ If you are looking for our old [legacy API documentation, see this](https://chec
 
 ### General
 
-Our payment processing service related documentation.
+Documentation for our payment processing service.
 
 <p class="tip">
 If you have any feedback regarding how we could improve the documentation, [please file an issue on Github](https://github.com/CheckoutFinland/psp-api/issues).
```
Comment (mikaturunen) [inline docs/README.md]: If you have any feedback and want us to improve the documentation, please file a issue on github.

File: docs/README.md [hunk 5]
```diff
@@ -20,56 +20,58 @@ Thank you!
 
 ### Test credentials
 
-Please note that not all payment methods support testing. The payment methods that support testing payments are enabled.
+Please note that not all payment methods support testing, so only the payment methods that support testing payments are enabled for these credentials.
 
 * Merchant Id: `375917`
```
Comment (mikaturunen) [inline docs/README.md]: Besides with the saippuakauppias, I can now use the api with merchant in test mode, right-o?

**Review by mikaturunen — COMMENTED** [review]
LGTM, I mostly paid attention to the way we present ourselves on the document.

**Comment by mikaturunen** [comment]
Well that was f'n fancy, commented on wrong commit.. Well, moved the comments to the right one then :D

**Pull request merged**